### PR TITLE
Better Classification using Fink filters

### PR DESCRIPTION
**Pull request by Theodlz**
### This PR changes the way the classification of alerts is made.
Before, we used a shortcut that was basically " a topic of alerts is a classification", but inside of a topic, many different objects with more detailed classifications can be found.
However, an alert from fink does not contain that classification, as it in the classification that was used to attribute it to a topic. However, what you can find in the alert is the "science", the data used to classify. Therefore, using `fink-filters`, you can recreate said classification.

This PR adds this aspect of the classification, and attempts to match fink classifications with classifications from a skyportal taxonomy by recursively inspecting every `class` and their `other names` field.

We also updated the logs in the console when running the script.

#### What more can be done:
We are waiting for modifications on fink-filters repo, as currently the module that contains the function that recreates the classification import pyspark and pyarrow, that combined take more than 350+ MB. However, they are actually not needed. An issue has been opened here: https://github.com/astrolabsoftware/fink-filters/issues/116

### CODE REVIEW

File: skyportal_fink_client/utils/skyportal_api.py [hunk 1]
```diff
@@ -961,24 +969,35 @@ def get_taxonomy_id_including_classification(classification: str, url: str, toke
     ----------
         status_code : int
             HTTP status code
-        exists : bool
-            True if the classification is in the hierarchy, False otherwise
+        exists : string
```
Comment (JulienPeloton) [inline skyportal_fink_client/utils/skyportal_api.py:972]: Nit: the name `exists` does not really reflect the meaning of the variable here (exists would rather be a boolean).

**Review by JulienPeloton — APPROVED** [review]
Thanks @Theodlz -- it looks good to me! I only left one minor comment, and the PR can be merged.

File: skyportal_fink_client/utils/skyportal_api.py (hunk 1 above)
Comment (Theodlz) [reply skyportal_fink_client/utils/skyportal_api.py]: Fixed ! I replaced the names of return arguments that dont have a declared variable (given name) by `out` as you suggested me to earlier.

**Comment by JulienPeloton** [comment]
Thanks @Theodlz !

**Pull request merged**